### PR TITLE
[Hotfix][E2E] Fix RabbitmqIT

### PR DESCRIPTION
**Pull request by hailin0**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by hailin0** [comment]
https://github.com/apache/incubator-seatunnel/actions/runs/4705553294/jobs/8346207289?pr=4536
image

**Comment by hailin0** [comment]
@531651225 @TyrantLucifer PTAL

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-rabbitmq-e2e/src/test/java/org/apache/seatunnel/e2e/connector/rabbitmq/RabbitmqIT.java [hunk 1]
```diff
@@ -66,6 +67,10 @@
 import java.util.Set;
 import java.util.stream.Stream;
 
+@DisabledOnContainer(
```
Comment (TyrantLucifer) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-rabbitmq-e2e/src/test/java/org/apache/seatunnel/e2e/connector/rabbitmq/RabbitmqIT.java:70]: revert

**Review by 531651225 — APPROVED** [review]
LGTM.

**Review by TyrantLucifer — APPROVED** [review]

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**